scipy/stats/_stats_py.py

```diff
@@ -6978,8 +6978,8 @@ def ttest_ind_from_stats(mean1, std1, nobs1, mean2, std2, nobs2,
     r"""
     T-test for means of two independent samples from descriptive statistics.
 
-    This is a test for the null hypothesis that two independent
-    samples have identical average (expected) values.
+    This is a test for the null hypothesis that two independent samples have
+    identical average (expected) values.
```

Comment:
```suggestion
    This is a test for the null hypothesis that two independent
    samples have identical average (expected) values.
```

Reply:
Removed that change in https://github.com/scipy/scipy/commit/8b60b333ccf2df714169909440e9a8273395f5e9.